index.html

```diff
@@ -10,7 +10,7 @@
 <header>
    <nav id="topbar">
       <ul>
-         <li><a href="#">HOME</a></li>
+         <li><a href="#" class="New Class">HOME</a></li>
```

Comment:
For CSS class targeting to work, we typically do not want any whitespace in the class name.
```suggestion
         <li><a href="#" class="new-class">HOME</a></li>
```